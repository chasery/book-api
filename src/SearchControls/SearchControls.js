import React from "react";
import SearchBar from "../SearchBar/SearchBar";
import FilterGroup from "../FilterGroup/FilterGroup";
import "./SearchControls.css";

class SearchControls extends React.Component {
  render() {
    return (
      <section className="SearchControls">
        <SearchBar />
        <FilterGroup />
      </section>
    );
  }
}

export default SearchControls;
