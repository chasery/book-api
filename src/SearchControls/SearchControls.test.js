import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import SearchControls from "./SearchControls";

describe("SearchControls component", () => {
  it("renders without error", () => {
    const wrapper = shallow(<SearchControls />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
