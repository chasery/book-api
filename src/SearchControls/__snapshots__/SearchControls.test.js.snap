// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchControls component renders without error 1`] = `
<section
  className="SearchControls"
>
  <SearchBar />
  <FilterGroup />
</section>
`;
